reference/enterprise-requirements.rst

```diff
@@ -58,5 +59,37 @@ applies:
 |                       |                       | services              |
 +-----------------------+-----------------------+-----------------------+
 
-[note] **Note:** Three nodes are required for multi-node operation.
-[/note]
+.. note ::
+    Three nodes are required for multi-node operation.
+
+Role Based Minimum Memory Sizing
```

Comment:
Do we have the data for CPU consumption? I guess we need a similar table for CPU modelling.

Reply:
That thought had occurred to me as well but no I don't have that; for Ceph we can go with the recommendations made for microceph/charmed ceph.